printer_mods/maramao/README.md

```diff
@@ -0,0 +1,14 @@
+# BTT piTFT43 v2 Mount
+
+this is a quick remix based on the work of alanho [BTT_PITFT50_v2_Mount] (https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/alanho/BTT_PITFT50_v2_Mount) 
+to fit on the screen mount of [sttts's Waveshare 5.5inch mount] (https://github.com/VoronDesign/VoronUsers/blob/master/printer_mods/sttts/Waveshare-5.5-inch-HDMI-AMOLED)
+to use the BTT piTFT43 
+
+## BOM
+4x M3x6mm
+4x M2.5x4mm (included in the BTT PITFT50 v2)
+500mm ~ 600mm Ribbon Flat Cable for Raspberry Pi Camera (depends on the position of the Raspberry Pi. The included one is too short)
+
+## Instrucions
+Use the mount in [sttts's mods] (https://github.com/VoronDesign/VoronUsers/blob/master/printer_mods/sttts/Waveshare-5.5-inch-HDMI-AMOLED/STLs/Voron-2.4-Mount-Generisch-v6.stl) to mount this to the frame.
```

Comment:
Remove space between [] and () and change to relative link:

    [sttts's mods](../../sttts/Waveshare-5.5-inch-HDMI-AMOLED/STLs/Voron-2.4-Mount-Generisch-v6.stl)